examples/post-meta-block/src/subtitle/block.json

```diff
@@ -1,6 +1,6 @@
 {
 	"$schema": "https://schemas.wp.org/trunk/block.json",
-	"apiVersion": 3,
+	"apiVersion": 2,
```

Comment:
why?

Reply:
This is for testing. I'm pulling these changes out now.